app/models/manageiq/providers/amazon/storage_manager/s3/cloud_object_store_container.rb

```diff
@@ -0,0 +1,10 @@
+class ManageIQ::Providers::Amazon::StorageManager::S3::CloudObjectStoreContainer < ::CloudObjectStoreContainer
+  include_concern 'Operations'
+
+  supports :delete
```

Comment:
As discussed on gitter, lets let it fail if e.g. there is no permission. We will fine-tune it later.